content/resources/workshop.md

```diff
@@ -537,6 +521,26 @@ Here is an example of customizations made to the default starter project using t
 
 </div>
 
+## Next Steps 
+
+You can continue reinforcing the concepts above by creating a new page in your Quire project. 
+
+<div class="try-it-yourself">
```

Comment:
Looks like this is missing it's closing `</div>` or I maybe I just can't see it in the diff